adapters/criteo/criteo_test.go

```diff
@@ -1,27 +1,15 @@
 package criteo
 
 import (
-	"testing"
-
 	"github.com/prebid/prebid-server/adapters/adapterstest"
-	"github.com/prebid/prebid-server/config"
-	"github.com/prebid/prebid-server/openrtb_ext"
+	"testing"
 )
 
 func TestJsonSamples(t *testing.T) {
 
 	// Setup:
-	bidder, buildErr := builderWithGuidGenerator(
-		openrtb_ext.BidderCriteo,
-		config.Adapter{
-			Endpoint: "https://bidder.criteo.com/cdb?profileId=230",
-		},
-		config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"},
-		newFakeGuidGenerator("00000000-0000-0000-00000000"),
-	)
-
-	if buildErr != nil {
-		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	bidder := &adapter{
+		endpoint: "https://bidder.criteo.com/cdb?profileId=230",
```

Comment:
just a suggestion. @vraybaud, can decide whether to make this change.

`https://bidder.criteo.com/cdb?profileId=230` is old endpoint url. should change this to one mentioned in bidder params `https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230`.  so that test setup and config is in sync.

Reply:
Hello @onkarvhanumante , thanks for the suggestion, I changed the endpoint url to mention the new one.

Reply:
@vraybaud 

tests are failing after ee01136 changes

```
--- FAIL: TestJsonSamples (0.00s)
    test_json.go:107: 
        	Error Trace:	/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:234
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:395
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:107
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:77
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:56
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/criteo_test.go:16
        	Error:      	Received unexpected error:
        	            	criteotest/exemplary/simple-banner-cookie-uid.json: httpRequest[0].uri "https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230" does not match expected "https://bidder.criteo.com/cdb?profileId=230."
        	Test:       	TestJsonSamples
        	Messages:   	criteotest/exemplary/simple-banner-cookie-uid.json Expected RequestData was not returned by adapters' MakeRequests() implementation: httpRequest[0]
    test_json.go:107: 
        	Error Trace:	/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:234
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:395
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:107
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:77
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:56
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/criteo_test.go:16
        	Error:      	Received unexpected error:
        	            	criteotest/exemplary/simple-banner-inapp.json: httpRequest[0].uri "https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230" does not match expected "https://bidder.criteo.com/cdb?profileId=230."
        	Test:       	TestJsonSamples
        	Messages:   	criteotest/exemplary/simple-banner-inapp.json Expected RequestData was not returned by adapters' MakeRequests() implementation: httpRequest[0]
    test_json.go:107: 
        	Error Trace:	/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:234
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:395
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:107
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:77
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:56
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/criteo_test.go:16
        	Error:      	Received unexpected error:
        	            	criteotest/exemplary/simple-banner-uid.json: httpRequest[0].uri "https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230" does not match expected "https://bidder.criteo.com/cdb?profileId=230."
        	Test:       	TestJsonSamples
        	Messages:   	criteotest/exemplary/simple-banner-uid.json Expected RequestData was not returned by adapters' MakeRequests() implementation: httpRequest[0]
    test_json.go:107: 
        	Error Trace:	/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:234
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:395
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:107
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:77
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:57
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/criteo_test.go:16
        	Error:      	Received unexpected error:
        	            	criteotest/supplemental/204-response-from-target.json: httpRequest[0].uri "https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230" does not match expected "https://bidder.criteo.com/cdb?profileId=230."
        	Test:       	TestJsonSamples
        	Messages:   	criteotest/supplemental/204-response-from-target.json Expected RequestData was not returned by adapters' MakeRequests() implementation: httpRequest[0]
    test_json.go:107: 
        	Error Trace:	/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:234
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:395
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:107
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:77
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:57
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/criteo_test.go:16
        	Error:      	Received unexpected error:
        	            	criteotest/supplemental/400-response-from-target.json: httpRequest[0].uri "https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230" does not match expected "https://bidder.criteo.com/cdb?profileId=230."
        	Test:       	TestJsonSamples
        	Messages:   	criteotest/supplemental/400-response-from-target.json Expected RequestData was not returned by adapters' MakeRequests() implementation: httpRequest[0]
    test_json.go:107: 
        	Error Trace:	/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:234
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:395
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:107
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:77
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:57
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/criteo_test.go:16
        	Error:      	Received unexpected error:
        	            	criteotest/supplemental/500-response-from-target.json: httpRequest[0].uri "https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230" does not match expected "https://bidder.criteo.com/cdb?profileId=230."
        	Test:       	TestJsonSamples
        	Messages:   	criteotest/supplemental/500-response-from-target.json Expected RequestData was not returned by adapters' MakeRequests() implementation: httpRequest[0]
    test_json.go:107: 
        	Error Trace:	/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:234
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:395
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:107
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:77
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/test_json.go:57
        	            				/home/runner/work/prebid-server/prebid-server/adapters/criteo/criteo_test.go:16
        	Error:      	Received unexpected error:
        	            	criteotest/supplemental/simple-banner-with-ipv6.json: httpRequest[0].uri "https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230" does not match expected "https://bidder.criteo.com/cdb?profileId=230."
        	Test:       	TestJsonSamples
        	Messages:   	criteotest/supplemental/simple-banner-with-ipv6.json Expected RequestData was not returned by adapters' MakeRequests() implementation: httpRequest[0]
FAIL
```

Reply:
Thank you for the review, I've updated the tests so they pass. 😄